platform/platform-app/pom.xml

```diff
@@ -321,44 +321,84 @@
+        <cxf.surefire.format>brief</cxf.surefire.format>
+        <cxf.surefire.usefile>false</cxf.surefire.usefile>
+        <cxf.surefire.parallel.mode />
+        <cxf.surefire.fork.vmargs>-ea</cxf.surefire.fork.vmargs>
+        <cxf.server.launcher.vmargs>-ea</cxf.server.launcher.vmargs>
+        <cxf.surefire.enable.assertions>true</cxf.surefire.enable.assertions>
+        <cxf.compile.flags>-Xlint:unchecked,deprecation,fallthrough,finally</cxf.compile.flags>
+        <cxf.compile.show.deprecation>true</cxf.compile.show.deprecation>
+        <cxf.codegenplugin.forkmode>once</cxf.codegenplugin.forkmode>
+        <cxf.codegen.jvmArgs />
+        <cxf.eclipse.outputDirectory>${basedir}/target/classes</cxf.eclipse.outputDirectory>
+        <!-- stuff related to ASM -->
+        <cxf.asm.groupId>org.ow2.asm</cxf.asm.groupId>
+        <cxf.asm.artifactId>asm</cxf.asm.artifactId>
+        <cxf.asm.version>5.0.4</cxf.asm.version>
+        <cxf.osgi.asm.version>[3.0,6.1)</cxf.osgi.asm.version>
+        <!-- OSGi related properties -->
+        <cxf.fragment.host />
+        <cxf.bundle.activator />
+        <cxf.osgi.import />
+        <cxf.osgi.symbolic.name>${project.groupId}.${project.artifactId}</cxf.osgi.symbolic.name>
+        <cxf.osgi.dynamic.import />
+        <cxf.osgi.spring.version>[3.1,5)</cxf.osgi.spring.version>
+        <cxf.osgi.spring.osgi.version>[1.2.0,2)</cxf.osgi.spring.osgi.version>
+        <cxf.osgi.commons.lang.version>[2.4,3)</cxf.osgi.commons.lang.version>
+        <cxf.osgi.slf4j.version>[1.5,2)</cxf.osgi.slf4j.version>
+        <cxf.osgi.javax.annotation.version>[0.0,2)</cxf.osgi.javax.annotation.version>
+        <cxf.osgi.javax.bind.version>[0.0,3)</cxf.osgi.javax.bind.version>
+        <cxf.osgi.javax.stream.version>[0.0,2)</cxf.osgi.javax.stream.version>
+        <cxf.osgi.javax.activation.version>[0.0,2)</cxf.osgi.javax.activation.version>
+        <cxf.osgi.javax.mail.version>[0.0,2)</cxf.osgi.javax.mail.version>
+        <cxf.osgi.javax.servlet.version>[0.0,4)</cxf.osgi.javax.servlet.version>
+        <cxf.osgi.javax.xml.ws.version>[0.0,3)</cxf.osgi.javax.xml.ws.version>
+        <!-- please maintain alphabetical order here -->
         <cxf.abdera.version>1.1.3</cxf.abdera.version>
-        <cxf.activemq.version>5.12.0</cxf.activemq.version>
+        <cxf.abdera.osgi.version>1.1.3_2</cxf.abdera.osgi.version>
+        <cxf.activemq.version>5.14.4</cxf.activemq.version>
         <cxf.ahc.version>1.9.8</cxf.ahc.version>
         <cxf.apacheds.version>2.0.0-M23</cxf.apacheds.version>
         <cxf.atmosphere.version>2.3.7</cxf.atmosphere.version>
-        <cxf.atmosphere.version.range>[2.0,3.0)</cxf.atmosphere.version.range>
+        <cxf.atmosphere.version.range>[2.0,2.4)</cxf.atmosphere.version.range>
         <cxf.axiom.version>1.2.14</cxf.axiom.version>
-        <cxf.bcprov.version>1.52</cxf.bcprov.version>
+        <cxf.bcprov.version>1.55</cxf.bcprov.version>
```

Comment:
If bouncy castle got updated to 1.55, we need to also update the bouncy castle libraries in the ddf /lib folder. Those are at 1.54 and we don't pull in the updated versions via the features.